src/preferences.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'unsafe-inline'">
+  <link rel="stylesheet" href="./assets/css/bulma.min.css">
+  <link rel="stylesheet" href="./assets/css/style.css">
+  <title>Preferences</title>
+</head>
+
+<body>
+
+  <header>
+
+      <nav class="navbar is-dark" role="navigation" aria-label="main navigation">
+        <div class="navbar-brand">
+          <div class="navbar-item">
+            <img src="./icons/streaker-icon.png" alt="Streaker Logo">
+          </div>
+          <div class="navbar-item">
+            <span>Streaker</span>
+          </div>
+        </div>
+      </nav>
+
+  </header>
+
+  <main>
+
+    <div class="field">
+      <label class="label">GitHub Username</label>
+      <div class="control">
+        <input id="githubUsername" type="text" class="input" placeholder="Type your GitHub username">
+      </div>
+    </div>
+
+    <div class="field">
+      <label class="label">GitHub Sync Interval</label>
+      <div class="control">
+        <input id="githubSyncInterval" type="number" class="input" placeholder="15">
+        <span class="input-text">Minutes</span>
+      </div>
+    </div>
+
+    <div class="field-checkbox">
+      <span>Launch at login</span>
```

Comment:
This should probably be capitalised.

```suggestion
      <span>Launch at Login</span>
```